### ISSUE TITLE
config about DeepLO

### ISSUE THREAD

**Issue by leyuan-sun** [ArashJavan/DeepLIO#20]
### DeepLIO Network ##############################
deeplio:
  dropout: 0.25
  pretrained: false
  model-path: ""
  lidar-feat-net:
    name: "lidar-feat-simple-1"
    pretrained: false
    model-path: ""
  imu-feat-net:
    name: "imu-feat-rnn"  # See the name is commented out
    pretrained: false
    model-path: ""
  odom-feat-net:
    name: # "odom-feat-fc" # See the name is commented out
    pretrained: false
    model-path: ""
  fusion-net: "fusion-layer"

why you comment `odom-feat-fc`, I think odom is still necessary but `fusion-net `should be commented.

**Comment by leyuan-sun** [comment]
I have trained lidarsegpoint net + odometry RNN net, the result is very bad no matter in training dataset and test dataset. Could you give me some instructions about how to reproduce the result of DeepLO in the paper?
Btw, in DeepVO (https://github.com/ChiWeiHsiao/DeepVO-pytorch), the network is CNN + LSTM, which I think is similar with lidarsegpoint net + odometry RNN net I have tested. But the results are very bad in my test.

**Comment by leyuan-sun** [comment]
I know why you did not comment fusion-net in DeepLO, because the fusion-net will be created only when both lidar-net and imu-net existed. So why you comment out odom-net in DeepLO?
